public/locales/en/translation.json

```diff
@@ -30,6 +30,9 @@
           }
         }
       },
+      "shareScenario": {
```

Comment:
it's weird to create a new entry here, all other labels are in commoncomponents.dialog.share.button

Reply:
Done :slightly_smiling_face: